data.sql

```diff
@@ -47,4 +47,41 @@ UPDATE animals
                       WHEN name IN ('Devimon', 'Plantmon') THEN (SELECT id FROM owners WHERE full_name = 'Bob' )
                       WHEN name IN ('Charmander', 'Squirtle', 'Blossom') THEN (SELECT id FROM owners WHERE full_name = 'Melody Pond' )
                       WHEN name IN ('Angemon', 'Boarmon') THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester' )
-                    END;
\ No newline at end of file
+                    END;
+
+/* Insert  data for vets TABLE: */
+INSERT INTO vets(name , age , date_of_graduation) VALUES
+                ('William Tatcher' , 45,  '2000-04-23'),
+                ('Maisy Smith' , 26, '2019-06-17', ),
+                ('Stephanie Mendez' , 64, '1981-05-04'),
+                ('Jack Harkness' , 38, '2008-06-08');
+
+/* Insert the data for specialties: */
+INSERT INTO specializations(vet_id, species_id) VALUES
+  ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon')),
+  ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon')),
+  ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon')),
+  ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
+
+/* Insert the following data for visits: */
+INSERT INTO visits ( animal_id, vet_id, visit_date) VALUES
+  ((SELECT id FROM animals WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24'),
+  ((SELECT id FROM animals WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22'),
+  ((SELECT id FROM animals WHERE name = 'Gabumon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-02'),
+  ((SELECT id FROM animals WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-06-05'),
```

Comment:
- [ ] I guess you made a mistake here. The first visit date for animal `Pikachu` to vet `Maisy Smith` is on `Jan 1st 2020` not `Jun 5 2020`. Let's update the date here.

Reply:
I think we both made a mistake because is on Jan 5th, 2020. :'D